pkg/operation/hybridbotanist/controlplane.go

```diff
@@ -260,8 +288,22 @@ func (b *HybridBotanist) DeployKubeScheduler() error {
 
 	schedulerConfig := b.Shoot.Info.Spec.Kubernetes.KubeScheduler
 	if schedulerConfig != nil {
-		defaultValues["featureGates"] = schedulerConfig.FeatureGates
+		featureGates = schedulerConfig.FeatureGates
```

Comment:
Put those checks in and not in `go` code

https://github.com/gardener/gardener/blob/92ca7287cf2cb7287410c6130fae4b32c9201dd4/charts/seed-controlplane/charts/kube-scheduler/templates/_helpers.tpl#L1-L5

Reply:
Done. Refer - https://github.com/gardener/gardener/pull/267/commits/062e57bda20c41dce228a5c4159f28b5eb766de3?utf8=%E2%9C%93&diff=unified#diff-a422a9d10a682f8e4a4c20051076b849R5